docs/commands/utter.md

```diff
@@ -28,39 +28,35 @@ The utter command will return the full request and response of the interaction w
 
 By default, the system will:
 
-* Use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
+* Use the Interaction Model in the models folder under the current working directory
+* If there's no Interaction Model, use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
 * Use the service currently running via the `bst proxy` command
 
 If no service is currently running via bst proxy, and HTTP endpoint can be specified with the `--url` option:
 ```
 $ bst utter Hello World --url https://my.skill.com/skill/path
 ```
 
-## Speech Asset Format and Location
-If your speech assets  (Intent Model and Sample Utterances) are not stored under ./speechAssets, you can use an option to specify another location.
+## Interaction Model Format and Location
+If your Interaction Model is not stored under ./models, or you have multiple locales, you can use an option to specify another location.
 
 By default, we look for:
 
-* `./speechAssets/IntentSchema.json`
-* `./speechAssets/SampleUtterances.txt`
+* `./models/en-US.json`
 
 Example:
 ```
-$ bst utter Hello World -i interactions/IntentSchema.json -s interactions/SampleUtterances.txt
+$ bst utter Hello World -m models/en-UK.json
 ```
 
-The format of these files is the same as they are entered in the Alexa Skill configuration.  
+The format of these file is the same as the one entered in the Alexa Skill configuration.
 
-The Intent Schema is a JSON file. Samples utterances is a space-delimited text file.
-
-An example of these files can be found [here](https://github.com/alexa/skill-sample-nodejs-hello-world/tree/master/speechAssets).
+The Interaction Model is a JSON file.
+An example of these file can be found [here](https://github.com/alexa/skill-sample-nodejs-fact/blob/en-US/models/en-US.json).
 
 ## Working With Slots
 
-Slot handling is a bit tricky. To send an utterance that uses slots, surround the slot variables like so:  
-```
-{MySlot}
-```
+Slot handling is transparent, we check your defined slots and samples and extract them. To send an utterance that uses slots, just write the utterance following one of your samples formats.
```

Comment:
"Slot handling is automatic - we check for defined slots and samples and extract them. To send an utterance that uses slots, just write it as you would say it."